### DISCUSSION TITLE
Get documentation link from the XML report

### DISCUSSION THREAD

**Discussion by mamupf** [detekt/detekt#7768] [Q&A]
Is there a easy and simple way to get or construct the documentation URL link from the Detekt XML report?

For example how to construct the documentation URL links?
- `source="detekt.FunctionNaming"` ->  https://detekt.dev/docs/rules/naming#functionnaming
- `source="detekt.EmptyFunctionBlock"` https://detekt.dev/docs/rules/empty-blocks#emptyfunctionblock

**Comment by cortinico** [comment]
> Is there a easy and simple way to get or construct the documentation URL link from the Detekt XML report?

Have you consider using the SARIF report instead? IIRC we add documentation links in that report

**Reply by mamupf** [reply]
Thanks you! That works for me.
